server/routes/validators/induction/additionalTrainingFormValidator.ts

```diff
@@ -65,7 +66,7 @@ const validateAdditionalTrainingOther = (
       errors.push(
         `Enter the type of training or vocational qualification ${prisonerSummary.firstName} ${prisonerSummary.lastName} has`,
       )
-    } else if (additionalTrainingOther.length > MAX_OTHER_LENGTH) {
+    } else if (textValueExceedsLength(additionalTrainingOther, MAX_OTHER_LENGTH)) {
```

Comment:
Changed all text field length validators (regardless of whether they might contain a CRLF or not) to use the new function `textValueExceedsLength` (defined below 👇 ) which takes account of any CRLFs in the content